src/template.json

```diff
@@ -40,21 +40,22 @@
   "provisioners": [
     {
       "ansible_env_vars": [
-        "ANSIBLE_SSH_ARGS='-o IdentitiesOnly=yes'"
+        "ANSIBLE_SSH_ARGS='-o IdentitiesOnly=yes'",
+        "SSH_USERNAME={{user `SshUsername`}}"
       ],
       "playbook_file": "playbook.yaml",
       "type": "ansible"
     }
   ],
   "variables": {
-    "AmiUsers": "563295687221,055273631518,804034162148,237179673806",
+    "AmiGroups": "all",
     "Department": "Platform",
     "ImageName": "{{env `ImageName`}}",
     "InstanceType": "t3.micro",
     "OwnerEmail": "tess.thyer@sagebase.org",
     "Project": "Infrastructure",
-    "SnapshotUsers": "563295687221,055273631518,804034162148,237179673806",
-    "SourceImage": "ami-05cce55c6aa92f55b",
+    "SnapshotGroups": "all",
+    "SourceImage": "ami-0d383722ce22a6ea1",
```

Comment:
don't we need a new packer-base-ubuntu-bionic release and update of ami id on this PR?

Reply:
Yes, but had to wait for it to build. Hopefully this third PR is the last alteration we make to the base image for SC-58.